### DISCUSSION TITLE
How to add a submit button to q-select autocomplete?

### DISCUSSION THREAD

**Discussion by maggie44** [quasarframework/quasar#14268] [General - Components / Directives / etc]
I am using the autocomplete select instead of an input box because I want to store previously submitted values in LocalStorage and provide them as options on return to the page. With an input box though I would usually be able to add a submit button next to it using templates. With the q-select how would be the best way to achieve the same effect?

Here is the select:

```
        <q-select
          v-model="model"
          placeholder="test.local"
          class="q-mt-md"
          :rules="[
          (value: string) =>
            !value.includes(' ') ||
            $t('components.wifi.configure_password.invalid_name')
        ]"
          :label="$t('general.hostname')"
          filled
          use-input
          multiple
          input-debounce="0"
          :options="filterOptions"
          style="width: 250px"
          @new-value="createValue"
          @filter="filterFn"
        >
```

I need to be able to call `createValue` and pass the `val` and `done` props from a template but `template #after="props"` doesn't contain anything, so when I submit createValue those functions are missing. 

https://quasar.dev/vue-components/select#example--text-autocomplete

**Comment by MilosPaunovic** [comment]
Try something like this:
https://codepen.io/milospaunovic/pen/zYjrdeG?editors=101
